spec/history.cddl

```diff
@@ -0,0 +1,58 @@
+can-history = tagged<{
+  total_num_changes : unsigned
+  changes : [* can-change]
+}>
+
+can-change = {
```

Comment:
nit: I suggest expanding `can` and say `canister` for clarity? Otherwise, it can be confusing, as it's not clear whether this means it's a "canister change" or if it's something that "can change".